src/app.jsx

```diff
@@ -5,6 +5,8 @@ const ipcRenderer = require('electron').ipcRenderer;
 const _ = require('underscore');
 const remote = require('electron').remote; // remote allows for using node modules within render process.
 const electronFs = remote.require('fs');
+const hyphyVision = require('hyphy-vision');
+require("style-loader!css-loader!sass-loader!hyphy-vision/dist/application.scss");
```

Comment:
Doesn't webpack take care of sass-loader? Do we need to change how we're deploying hyphy-vision's scss to help?